lib/lambda-cfn.js

```diff
@@ -24,6 +24,28 @@ lambdaCfn.lambdaSnsUser = lambdaSnsUser;
 lambdaCfn.lambdaSnsUserAccessKey = lambdaSnsUserAccessKey;
 lambdaCfn.outputs = outputs;
 lambdaCfn.load = load;
+lambdaCfn.getEnv = getEnv;
+
+function stripHyphens(r) {
+    return r.replace(/-/g, '');
+}
+
+var packages;
+if (process.env.NODE_ENV == 'test') {
+  packages = require(path.join(root,'/test/package.json'));
+} else {
+  packages = require(path.join(root,'package.json'));
+}
+
+function namespace(name,path) {
```

Comment:
@zmully you _might_ be able to use module.children - globally available - here if you're looking for a way to do this w/o reaching out to package.json.